fvm/evm/types/block.go

```diff
@@ -87,15 +89,37 @@ func NewBlockFromBytes(encoded []byte) (*Block, error) {
 	return res, nil
 }
 
-// GenesisBlock is the genesis block in the EVM environment
-var GenesisBlock = &Block{
-	ParentBlockHash: gethCommon.Hash{},
-	Height:          uint64(0),
-	TotalSupply:     new(big.Int),
-	ReceiptRoot:     gethTypes.EmptyRootHash,
+// GenesisTimeStamp returns the block time stamp for EVM genesis block
+func GenesisTimeStamp(flowChainID flow.ChainID) uint64 {
+	// default evm chain ID is previewNet
+	switch flowChainID {
+	case flow.Mainnet:
+		return uint64(time.Date(2024, time.August, 1, 0, 0, 0, 0, time.UTC).Unix())
+	case flow.Testnet:
+		return uint64(time.Date(2024, time.September, 1, 0, 0, 0, 0, time.UTC).Unix())
```

Comment:
Are the months here potentially swapped?

Reply:
it was a mistake, fixed it.